src/core_mapfiles/ecl.rs

```diff
@@ -173,17 +175,17 @@ static ECL_06: &'static CoreSignatures = &CoreSignatures {
         (Th06, 94, Some("")),
         (Th06, 95, Some("EfffssS")),
         (Th06, 96, Some("")),
-        (Th06, 97, Some("S")),
-        (Th06, 98, Some("ssssS")),
-        (Th06, 99, Some("SS")),
-        (Th06, 100, Some("S")),
+        (Th06, 97, Some("N")),
+        (Th06, 98, Some(r#"s(enum="AnmScript")s(enum="AnmScript")s(enum="AnmScript")s(enum="AnmScript")N"#)), // zero: s(enum="AnmScript")s(enum="AnmScript")s(enum="AnmScript")s(enum="AnmScript")s(enum="AnmScript")--
```

Comment:
What's up with this comment?

Reply:
Oh, padding.

Reply:
Those - characters represent padding bytes. It's the same as the other comments about technically incorrect signatures, just long thanks to the final "N" being replaced with "s(enum="AnmScript")--"